app/views/tournaments/show.html.erb

```diff
@@ -1,8 +1,46 @@
-<h1><%= @tournament.location %></h1>
-<h3>Director: <%= @tournament.user.email %></h3>
-<h4>Date: <%= @tournament.date.strftime('%B %e, %Y') %></h4>
-<p>Number of rounds: <%= @tournament.rounds %></p>
-<p>Number of winners:
-  <%= @tournament.number_of_winners.nil? ? "unknown" : @tournament.number_of_winners %>
-</p>
-<p>Pairing system: <%= @tournament.pairing_system.capitalize %></p>
+<div class="container">
+  <h1 class="mb-3 border-bottom border-3"><%= @tournament.location %></h1>
+
+  <div class="mb-3 border-bottom border-3">
+    <h4>Director: <%= @tournament.user.email %></h4>
+    <p>Date: <%= @tournament.date.strftime('%B %e, %Y') %></p>
+    <p>Number of rounds: <%= @tournament.rounds %></p>
+    <p>Number of winners:
+      <%= @tournament.number_of_winners.nil? ? "unknown" : @tournament.number_of_winners %>
+    </p>
+    <p>Pairing system: <%= @tournament.pairing_system.capitalize %></p>
+  </div>
+
+  <div class="mb-3 border-bottom border-3">
+    <p>Players</p>
+    <ul>
+      <% @tournament.players.each do |player| %>
```

Comment:
Looks good, but I have not yet scraped xtables to get all players of a tournament, so this will break

Reply:
This will work with my seed tournament